design-documents/graph-ql/coverage/b2b/negotiableQuotes.graphqls

```diff
@@ -174,12 +196,9 @@ input CloseNegotiableQuotesInput {
 }
 
 type CloseNegotiableQuotesOutput {
-    closed_quotes(
-        filter: NegotiableQuoteFilterInput,
-        pageSize: Int = 20,
-        currentPage: Int = 1
-        sort: NegotiableQuoteSortInput
-    ): NegotiableQuotesOutput @doc(description: "Quotes that were just closed")
+    result_status: TriStateMutationResultStatus
+    operation_results: [NegotiableQuoteOperationResult!]!
+    closed_quotes: [NegotiableQuote!] @doc(description: "An array containing the negotiable quotes that were just closed") @deprecated(reason: "Replaced with operation_results")
```

Comment:
`closed_quotes: [NegotiableQuote!]` is the actual schema in source (which didn't get updated here), and it should now be deprecated in favor of the new `operation_results` field.

Reply:
ok, it seems that it's the desired approach as verbally discussed to go further, so consider this comment as a sign off